visualization-design.typ

```diff
@@ -0,0 +1,36 @@
+= Visualization Design
+
+The visualization serves a dual purpose: To communicate the functionality of the load balancer at a glance to a non-technical audience, and to serve as a useful debugging tool during development.
+
+== Hosting
+
+Option 1: Self-Hosted
+- Pros: No added cost, no compromises required   
+- Cons: Not always online, longer startup time
+
+Option 2: Buy a domain
+- Pros: Always online, easy access
+- Cons: Cost (that's probably enough of a con)
+
+Option 3: Vercel
+- Pros: Easy, free
+- Cons: .vercel in the hostname, ???
```

Comment:
Why would we host on vercel when we are already hosting on fly?

Reply:
I might have misread but I thought you wrote somewhere that you'd prefer the visualization not be on fly.io

Reply:
Yeah, what I meant by that is that is we would just run it on our laptops

Reply:
The reason being that it would be less admin work to deal with. I already posted some of my thoughts in #224 for how this could be accomplished